plugins/genesys/fu-genesys-scaler-device.c

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2021 Gaël PORTAY <gael.portay@collabora.com>
+ * Copyright (C) 2021-2022 Gaël PORTAY <gael.portay@collabora.com>
```

Comment:
in general we only list the start year in source files

Reply:
done.